### PR TITLE
Add Nosto client script load spec

### PR DESCRIPTION
**Pull request by nosto-mikpou**
Adds Nosto client script and Shopify Markets script load test spec. This should increase more coverage whether the scripts are loaded properly.

### CODE REVIEW

File: spec/nosto.load.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { NostoProvider, NostoHome } from "../src/index"
+
+describe("Nosto client script loading", () => {
+  it("is stopped via provider when the script is loaded externally", async () => {
+    window.nosto = {
+      reload: vi.fn()
+    }
+
+    render(
+      <NostoProvider account="shopify-11368366139" loadScript={false}>
```
Comment (timowestnosto) [inline spec/nosto.load.spec.tsx:13]: is loadScript already available?

Comment (nosto-mikpou) [reply spec/nosto.load.spec.tsx:13]: True, I thought it was already available in master.

File: spec/nosto.load.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { vi, describe, expect, it } from "vitest"
+import { render } from "@testing-library/react"
+import { NostoProvider, NostoHome } from "../src/index"
+
+describe("Nosto client script loading", () => {
+  it("is stopped via provider when the script is loaded externally", async () => {
```
Comment (timowestnosto) [inline spec/nosto.load.spec.tsx:7]: async needed?

File: spec/nosto.load.spec.tsx (hunk 1 above)
Comment (nosto-mikpou) [reply spec/nosto.load.spec.tsx:13]: I'll add the test case as it.todo

**Review by timowestnosto — APPROVED** [review]

**Pull request merged**